README.md

```diff
@@ -242,6 +242,9 @@ https_cert_file = ""
 // The client key file to use for talking to HTTPS checks.
 https_key_file = ""
 
+// Metrics interface address
+metrics_interface = "127.0.0.1:8080"
```

Comment:
`metrics_interface` is the usual name we use for this at work, but since this endpoint may gain further features I changed it to your suggestion. Also corrected default value in documentation.

Reply:
Thanks for the context on `metrics_interface`. Thanks for changes here!